src/SFML/Graphics/Shape.cpp

```diff
@@ -204,7 +204,6 @@ void Shape::update()
     // Texture coordinates
     updateTexCoords();
 
-    // Outline
```

Comment:
True, but now it's inconsistent, because there's a comment above `updateTexCoords()`, but not above `updateOutline()`. In general, I'd focus more on the issue and less on all the other existing code :wink: